ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/operations/SignedBlsToExecutionChange.java

```diff
@@ -35,6 +35,9 @@
     super(type, backingNode);
   }
 
+  public static final SignedBlsToExecutionChangeSchema SSZ_SCHEMA =
+      new SignedBlsToExecutionChangeSchema();
```

Comment:
We need to get this from schema definitions - the old pattern was to have a constant for schemas, but that doesn't support data structures changing with hard forks so we've moved away from it to everything coming from `SchemaDefinitions` which is milestone aware.

Reply:
I'll push a commit to change this - easier than explaining it in a comment. :)

Reply:
I understand. Thanks! 🙏